### ISSUE TITLE
Week 4 Update

### ISSUE THREAD

**Issue by SidhantPuntambekar** [SidhantPuntambekar/BMI702_Final_Project#2]
README commit: https://github.com/SidhantPuntambekar/BMI702_Final_Project/commit/7f2d870259cdb8189af109f04759b42309ebcce2

<img width="809" height="724" alt="Image" src="https://github.com/user-attachments/assets/94a98b79-9722-4879-8ef8-4759b1c25320" />

What we did this week: 

- Went to office hours with Professor Zitnik (2/11) to discuss antibody prediction idea, also discussed generative model ideas such as flow metric models instead of diffusion models for this task
- Read papers on pre-trained antibody transformer models (AntiBERTy) and generative language models for antibody design (IgLM)
- Started (2/15) and continued writing proposal for submission due 2/20

What's next:

- Finish writing proposal for submission on 2/20
- Begin aggregating antibody sequence data for baseline modeling with XGBoost models/random forest models.
